src/sentry/issues/grouptype.py

```diff
@@ -18,6 +28,22 @@ class GroupCategory(Enum):
 _group_type_registry: Dict[int, Type[GroupType]] = {}
 _slug_lookup: Dict[str, Type[GroupType]] = {}
 _category_lookup: Dict[int, Set[int]] = defaultdict(set)
+DEFAULT_IGNORE_LIMIT: int = 3
+DEFAULT_EXPIRY_TIME: int = 60 * 60 * 24
+
+
+@dataclass(frozen=True)
+class NoiseConfig:
+    ignore_limit: int = DEFAULT_IGNORE_LIMIT
+    expiry_time: int = DEFAULT_EXPIRY_TIME
```

Comment:
Let's note that this is in seconds. Could make sense to convert it to a `timedelta` so that there's no potential for confusion, then just use `.total_seconds` later on.

Reply:
Ah yeah. Will comment